### ISSUE TITLE
frio: Please make Javascript URL preview optional

### ISSUE THREAD

**Issue by hoergen** [friendica/friendica#6958]
This Javascript URL Preview function is annoying me like hell. 
Therefore I would ask you **please** make it optional.

Reasons:
1. On my notbook screen I only have 3 lines to write https://github.com/friendica/friendica/issues/6721
2. I cant edit the Info text on the embeded url preview, because i can't scroll down 
3. It produces wrong code from time to time and there is then a trailing [/url] floating around
4. I can't decide anymore if I want to use preview image or full size image.
5. Its only available on posts not on comments. I know there was a discussion, but I still fully disagree to take away the freedom to decide it I just want to send a textlink or a fully embeded URL #continuitiy 

Please make it optional. Thank you.

### Friendica version you encountered the problem
Friendica develop / git / PHP7.2.15-1 / mariadb Ver 15.1 Distrib 10.1.37

**Comment by tobiasd** [comment]
part of #2552

**Comment by hoergen** [comment]
Can't find that point there.

**Comment by tobiasd** [comment]
I just referenced that PR here, so this Issue appears in the threat of the list, should anybody take the thread as guide for stuff that need to be addresses.

**Comment by tobiasd** [comment]
I just had the very annoying experience that there was a preview created, containing absolutely nothing after I pasted some URL into the editor. But the disfunctional attachment code was then send nevertheless.

I edited the posting after sending, so those networks that support edits in postings don't have the quirk included. But this hit me by surprise.

**Comment by hoergen** [comment]
Maybe this feature should be removed until it can be made optional AND it is working .

**Comment by hoergen** [comment]
Pong

**Comment by annando** [comment]
The automatically generated text can be edited. I'm not sure if this was always the case, I just realized some time ago.

**Comment by hoergen** [comment]
Yes you can edit it, IF your screen is big enough (like I wrote in https://github.com/friendica/friendica/issues/6721 ), but you can't disable it.

**Comment by hoergen** [comment]
This is still annoying when you edit your post or pasting several links in your post, every time you have to click that preview away. Please insert a switch where you can deactivate that annoying script. THANK YOU.

**Comment by hoergen** [comment]
Wontfix.

**Comment by MrPetovan** [comment]
> This is still annoying when you edit your post or pasting several links in your post, every time you have to click that preview away.

My current workaround is paste all the links first, and then clicking the preview away only once.
